pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -46,6 +46,11 @@ const (
 	CredentialPrefix = "credential"
 )
 
+const (
+	CCMMigrationNeededAnnotation = "ccm-migration.k8c.io/migration-needed"
+	CSIMigrationNeededAnnotation = "csi-migration.k8c.io/migration-needed"
```

Comment:
What is the lifecycle of those two annotations? Once the migration is fully-complete, is it safe to delete those two annotations, or those should stay for the whole lifecycle of the cluster?

Reply:
yes, they don't harm but that's the idea ;-)

Reply:
What's the idea actually? That it should be possible to delete those or no? :sweat_smile: